### PR TITLE
Center store icons

### PR DESCRIPTION
**Pull request by huwshimi**
Fixes: #2893.

### CODE REVIEW

**Comment by huwshimi** [comment]
QA:
- open the store
- resize your window to large and small sizes
- check that the icons in the large coloured blocks remain centered at all sizes

**Comment by jujugui** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://ci-cge.jujugui.org:8080//job/juju-gui/7339/

File: jujugui/static/gui/src/app/components/store/_store.scss [hunk 1]
```diff
@@ -92,41 +92,38 @@
...
   .box {
+    min-height: 310px;
     margin-bottom: 20px;
+    padding-bottom: 125px;
 
     &--kubernetes {
       overflow: hidden;
-      padding-bottom: 125px;
-      background-color: #3371e3;
+      background: #3371e3 url(https://assets.ubuntu.com/v1/2847e80a-k8-image.png) no-repeat center -50px;
```
Comment (anthonydillon) [inline jujugui/static/gui/src/app/components/store/_store.scss]: Do we no longer need all assets local. Originally this was to support offline usage.

File: jujugui/static/gui/src/app/components/store/_store.scss [hunk 2]
```diff
@@ -92,41 +92,38 @@
...
+      background-size: 350px;
```
Comment (anthonydillon) [inline jujugui/static/gui/src/app/components/store/_store.scss]: With this scale with the screen width? This should work on a mobile screen too.

File: jujugui/static/gui/src/app/components/store/_store.scss (hunk 1 above)
Comment (huwshimi) [inline jujugui/static/gui/src/app/components/store/_store.scss]: Good call, I've moved these locally.

File: jujugui/static/gui/src/app/components/store/_store.scss (hunk 2 above)
Comment (huwshimi) [inline jujugui/static/gui/src/app/components/store/_store.scss]: It doesn't scale, but it does seem to look fine at mobile sizes as well.

**Comment by jujugui** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://ci-cge.jujugui.org:8080//job/juju-gui/7345/

**Comment by jujugui** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://ci-cge.jujugui.org:8080//job/juju-gui/7348/

**Comment by hatched** [comment]
It looks like something is off centre with Kubernetes (OSX Chrome 58.0)

screen shot 2017-05-15 at 11 11 29 am

**Comment by huwshimi** [comment]
@hatch, ah, the image is centered, but the white block is not in the center of the image, I'll update it to not be centered :)

**Comment by huwshimi** [comment]
@hatched updated.

**Comment by jujugui** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://ci-cge.jujugui.org:8080//job/juju-gui/7351/

**Comment by huwshimi** [comment]
Thanks everyone :shipit:

**Comment by jujugui** [comment]
Status: merge request accepted. Url: http://ci-gce.jujugui.org:8080/job/juju-gui-merge

**Pull request merged**